packages/shared/lib/services/endUser.service.ts

```diff
@@ -158,3 +158,16 @@ export async function linkConnection(db: Knex, { endUserId, connection }: { endU
 export async function unlinkConnection(db: Knex, { connection }: { connection: Pick<StoredConnection, 'id'> }) {
     await db<StoredConnection>('_nango_connections').where({ id: connection.id! }).update({ end_user_id: null });
 }
+
+export async function getEndUserByConnectionId(db: Knex, props: { connectionId: number }): Promise<Result<EndUser, EndUserError>> {
+    const endUser = await db<DBEndUser>(END_USERS_TABLE)
```

Comment:
do we need the generic here since it is specified by the select below?

Reply:
true